netlogo-gui/src/main/app/App.scala

```diff
@@ -420,7 +421,8 @@ class App extends
                                  pico.getComponent(classOf[ExternalFileManager]))
 
     frame.addLinkComponent(_tabManager)
-
+    notificationBanner = new NotificationBanner()
+    frame.add(notificationBanner, java.awt.BorderLayout.NORTH)
```

Comment:
I was following the format for how tabManager was added a few lines below, but will change the import:
frame.getContentPane.add(_tabManager.mainTabs, java.awt.BorderLayout.CENTER)

Reply:
Unfortunately, there is a lot of code that hasn't been touched in a while, so not everything follows the style guide. I often try to fix old stuff that I find that doesn't fit the style guide, but it's more important to ensure that new code follows the style guide rather than worrying about old code. Aside from that issue, it is generally a good idea to copy from other parts of the repo, so it's good that you did that anyway.

Reply:
That change has been now made